app/src/main/java/com/kazakov/newyou/app/view/component/base/impl/view/PredictionView.java

```diff
@@ -41,9 +61,188 @@ public boolean onOptionsItemSelected(MenuItem item) {
+        createActivityInfoCell(p.getActivity().toString(), tableRow);
+        tableRow.addView(getDisabledTextViewForTableRow(p.getDuration(), tableRow));
+        createNumberOfRepeatCell(Integer.toString(p.getNumber_of_repeats()), tableRow);
+        return tableRow;
+    }
+
+    private TextView getDisabledTextViewForTableRow(String text, TableRow row) {
+        TextView editText = new TextView(row.getContext());
+        editText.setText(text);
+        editText.setEnabled(false);
+        editText.setWidth(cellSize);
+        editText.setTextAlignment(View.TEXT_ALIGNMENT_CENTER);
+        editText.setTextColor(Color.BLACK);
+        return editText;
+    }
+
+    private void createActivityInfoCell(String text, TableRow row) {
+
+        String[] texts = activityToStringArray();
+        TextSwitcher simpleTextSwitcher = createSimpleTextSwitcher(row.getContext());
+
+        simpleTextSwitcher.setCurrentText(text);
+        Button btnNext = new Button(row.getContext());
+        btnNext.setEnabled(false);
+        btnNext.setBackgroundResource(R.drawable.turn_right);
+        btnNext.setOnClickListener(v -> moveTextRight(texts, simpleTextSwitcher));
+        Button btnPrev = new Button(row.getContext());
+        btnPrev.setBackgroundResource(R.drawable.turn_left);
+
+        btnPrev.setOnClickListener(v -> moveTextLeft(texts, simpleTextSwitcher));
+        row.addView(btnPrev);
+        row.addView(simpleTextSwitcher);
+        row.addView(btnNext);
+        btnPrev.setVisibility(View.INVISIBLE);
+        btnNext.setVisibility(View.INVISIBLE);
+
+    }
+
+    private void createNumberOfRepeatCell(String text, TableRow row) {
+        TextSwitcher textSwitcher = createSimpleTextSwitcher(row.getContext());
+        textSwitcher.setCurrentText(text);
+        Button btnPlus = new Button(row.getContext());
+        btnPlus.setBackgroundResource(R.drawable.turn_right);
+        btnPlus.setVisibility(View.INVISIBLE);
+        btnPlus.setOnClickListener(v -> setIncreasedNumbersOfRepeats(textSwitcher));
+        Button btnMinus = new Button(row.getContext());
+        btnMinus.setBackgroundResource(R.drawable.turn_left);
+        btnMinus.setVisibility(View.INVISIBLE);
+        btnMinus.setOnClickListener(v -> setDecreasedNumbersOfRepeats(textSwitcher));
+        row.addView(btnMinus);
+        row.addView(textSwitcher);
+        row.addView(btnPlus);
+    }
+
+    public TextSwitcher createSimpleTextSwitcher(Context context) {
+        TextSwitcher simpleTextSwitcher = new TextSwitcher(context);
+        simpleTextSwitcher.setInAnimation(AnimationUtils.loadAnimation(context,
+                android.R.anim.fade_in));
+        simpleTextSwitcher.setOutAnimation(AnimationUtils.loadAnimation(context,
+                android.R.anim.fade_out));
+        simpleTextSwitcher.setFactory(() -> {
+            TextView t = new TextView(context);
+            t.setGravity(Gravity.TOP | Gravity.CENTER_HORIZONTAL);
+            t.setTextSize(textSize);
```

Comment:
please avoid using not single line lambda body, can replace it with method reference